requirements.txt

```diff
@@ -6,4 +6,4 @@ djangorestframework>=3.8.2,<3.10.0
 passlib>=1.7.1,<1.8.0
 django-filter>=2.1.0,<2.2.0
 djangorestframework-link-header-pagination>=0.1.1,<0.2.0
-xhtml2pdf>=0.2.3,<0.3.0
+weasyprint>=43,<=44 
```

Comment:
The latest weasyprint release is 50, why are you putting 44 as the upper bound?
https://pypi.org/project/WeasyPrint/#history